### DISCUSSION TITLE
How to edit the biochemical characteristics of surface organic matter?

### DISCUSSION THREAD

**Discussion by Malitoz** [APSIMInitiative/ApsimX#9220] [Q&A]
Hello, how can I edit the chemical characteristics of SOM; tithonia and fym for my case in ApisimX?

**Comment by Malitoz** [comment]
> Hello, how can I edit the chemical characteristics of SOM; tithonia and fym for my case in ApisimX?

@hol353 , @lie112 & @LenLon

**Comment by hol353** [comment]
Currently you cannot add new, or modify existing, Surface Organic Matter types from the user interface. The only way to do it would be to manually modify a file (SurfaceOrganicMatter.json) and then recompile APSIM.

**Reply by Malitoz** [reply]
> Currently you cannot add new, or modify existing, Surface Organic Matter types from the user interface. The only way to do it would be to manually modify a file (SurfaceOrganicMatter.json) and then recompile APSIM.

Where is this file located?

**Reply by hol353** [reply]
File is here: https://github.com/APSIMInitiative/ApsimX/blob/master/Models/Resources/SurfaceOrganicMatter.json. To edit the file, you'll need some tools like GIT and Visual Studio or Visual Studio Code and you'll need some knowledge on how to use them. It isn't a trivial process.
